Unicode/utype.c

```diff
@@ -41001,7 +41001,7 @@ const uint32 ____utype[] = { 0,
   0x180c2010, 0x00080000, 0x00004000, 0x00004000, 0x00004000, 0x00004000, 0x00000000, 0x00000000,
   0x10000000, 0x00000000, 0x10200100, 0x00000000, 0x00000000, 0x00020000, 0x00000000, 0x10000000,
   0x00004000, 0x00004000, 0x10000400, 0x10000400, 0x10010000, 0x10200101, 0x00000000, 0x00000000,
-  0x10000000, 0x10000400, 0x10200100, 0x00000000, 0x10000000, 0x10000000, 0x10000000, 0x00080000,
+  0x10000000, 0x10000400, 0x10200100, 0x00000000, 0x30000000, 0x30000000, 0x30000000, 0x00080000,
```

Comment:
On August 25, 2016 08:14:33 AM Frank Trampe wrote:
> > @@ -41001,7 +41001,7 @@ const uint32 ____utype[] = { 0,
> So this table is a bit-per-field mask, and we're interested in more
> codepoints than before.

yes.
For this PR, we're interested in avoiding a bsearch if we already know 
there is no such character {0..MAXC-1}
There are very few values above MAXC that are lig|vulg|frac, so we can 
probably tolerate a simple min<x<max before bsearching those higher values.

inc/utype.h has 32bit_field allotted for flag definitions. I shared 1 bit for 
{lig&vulg&frac} and there is still an unused flag bit that could be used for 
something else 0x4000000. After that, we'd have to build a 40bit flag or 
larger for more bits.